src/Login/redux/actions/LoginAction.js

```diff
@@ -0,0 +1,25 @@
+import { toast } from 'react-toastify';
+import axios from 'axios';
+import { loginTypes } from '../types';
+import { loginUrl } from '../loginUrls';
+
+// eslint-disable-next-line arrow-parens
```

Comment:
Fix this. I don't think it's hard fix.

Reply:
Well @AGMETEOR  l have fixed this as well